actions.yaml

```diff
@@ -17,6 +17,12 @@ list-backups:
   description: Lists backups in s3 storage.
 pre-upgrade-check:
   description: Run necessary pre-upgrade checks and preparations before executing a charm refresh.
+promote-cluster:
+  description: Promotes the cluster of choice to a primary cluster. Must be ran against the leader unit.
+  params:
+    force-promotion:
+      type: boolean
+      description: Force the promotion of a cluster when there is already a primary cluster.
```

Comment:
AS discussed, force promotion should be used for Failover. The safe switchover do not need force.

P.S. the very first join, when two are Primary can be done though Force to be safe.

Reply:
Sure. I created the following ticket for that: https://warthogs.atlassian.net/browse/DPE-4257